Fail to install due to google-closure-deps@20191027.0.1

cause: https://github.com/google/closure-library/issues/996

```console
npm i -D @teppeis/duck
npm ERR! code ETARGET
npm ERR! notarget No matching version found for google-closure-compiler@^v20191027.0.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'google-closure-deps'
npm ERR! notarget 
```

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "faastjs": "^3.0.13",
     "fastify": "^2.10.0",
     "glob": "^7.1.6",
-    "google-closure-deps": ">=20190325.0.0",
+    "google-closure-deps": ">=20190325.0.0 <=20191027.0.0",
     "listr": "^0.14.3",
     "merge-options": "^2.0.0",
     "p-limit": "^2.2.1",

